cfgov/jinja2/v1/_includes/atoms/select.html

```diff
@@ -0,0 +1,60 @@
+{# ==========================================================================
+
+   select.render()
+
+   ==========================================================================
+
+   Builds a select for a form.
+
+   settings:
+
+   label:         Name of the field.
+
+   placeholder:   Text for the placeholder.
+
+   fields:        An array of all the fields to select.
+
+   size:          The size of the field. Defaults to 1 (full-width).
+                  Possible options: 1, 1-2, 1-3
+
+   required:      Whether the field is required. Defaults to false.
+
+   disabled:      Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% if not value %}
+    {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+    {% do global_dict.update({
+        'select': {
+            'label':        'This is a label',
+            'placeholder':  'This is a placeholder',
+            'fields':       ['Option 1', 'Option 2', 'Option 3'],
+            'size':         '1',
+            'required':     true,
+            'disabled':     false
+        }
+    }) %}
+    {% set value = global_dict.select %}
+{% endif %}
+{% from 'macros/util/format/url.html' import slugify as slugify %}
+{% set id_slug = slugify( value.label ) %}
+<div class="form-l_col form-l_col-{{ value.size }}">
```

Comment:
All the values except "required" and "disabled" are required and shouldn't be missing. Do we still want to to include those checks?